dm/dm.go

```diff
@@ -71,13 +72,15 @@ var usage = func() {
 	fmt.Fprintln(os.Stderr, "Flags:")
 	flag.PrintDefaults()
 	fmt.Fprintln(os.Stderr)
-	os.Exit(1)
+	fmt.Fprintln(os.Stderr, "--stdin requires a tar archive of input files and a top level file name.")
+	fmt.Fprintln(os.Stderr)
+	panic("\n")
 }
 
 func getGitRegistry() *registry.GithubRegistry {
 	s := strings.Split(*template_registry, "/")
 	if len(s) < 2 {
-		log.Fatalf("invalid template registry: %s", *template_registry)
+		panic(fmt.Sprintf("invalid template registry: %s", *template_registry))
```

Comment:
Yeah, I figured that was the case after I read the code below, but as soon as a comment is made, it sends it rather than batching them. The bigger question I think is, if we use the tar library seems like we wouldn't have to have all these unwindings of the stack / recover? As in, use a library to untar instead of forking a process for it.